api/src/main/java/org/openmrs/util/DatabaseUpdater.java

```diff
@@ -675,8 +677,8 @@ public static void writeUpdateMessagesToFile(String text) {
 		try {
 			String lineSeparator = System.getProperty("line.separator");
 			Date date = Calendar.getInstance().getTime();
-			
-			writer = new PrintWriter(new BufferedWriter(new FileWriter(destFile, true)));
+
+			writer = new PrintWriter(new BufferedWriter(new OutputStreamWriter(new FileOutputStream(destFile, true), StandardCharsets.UTF_8)));
```

Comment:
I'm going to approve this for now. But it might be better to separate some of these into two lines though.

Eg :-
```
Reader reader = new InputStreamReader(new FileInputStream(fileName), StandardCharsets.UTF_8);
BufferedReader br = new BufferedReader(reader);
```
@dkayiwa will let you know if that is required :)

Reply:
Yes that would be clearer.
I also think that if he did mvn clean install, that long line would be reformatted to wrap to the next line.

Reply:
Fixed :)